src/Scripting/CSharp/Hosting/CommandLine/Csi.cs

```diff
@@ -18,15 +18,21 @@ internal CSharpInteractiveCompiler(string responseFile, string baseDirectory, st
         {
         }
 
+        internal override Type Type => typeof(CSharpInteractiveCompiler);
+
+        internal override string GetAssemblyFileVersion()
+        {
+            return Type.GetTypeInfo().Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>().Version;
```

Comment:
I don't have sufficient confidence that I fully understand why things are this way. I'd rather not document what is just a (possibly accurate) guess.